NOTICE.md

```diff
@@ -0,0 +1,677 @@
+[//]: # " Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved. "
+[//]: # "  "
+[//]: # " This program and the accompanying materials are made available under the "
+[//]: # " terms of the Eclipse Public License v. 2.0 which is available at "
+[//]: # " http://www.eclipse.org/legal/epl-2.0, "
+[//]: # " or the Eclipse Distribution License v. 1.0 which is available at "
+[//]: # " http://www.eclipse.org/org/documents/edl-v10.php. "
+[//]: # "  "
+[//]: # " SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause "
+
+# Notices for EclipseLink
+
+This content is produced and maintained by the EclipseLink project.
+
+* Project home: https://projects.eclipse.org/projects/ee4j.eclipselink
+
+## Trademarks
+
+ EclipseLink is a trademark of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License v1.0
+which is available at https://www.eclipse.org/org/documents/edl-v10.php.
+
+SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+
+## Source Code
+
+The project maintains the following source code repositories:
+
+* https://github.com/eclipse-ee4j/eclipselink-asm
```

Comment:
all our repos should be listed here, see https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.eclipselink which can generate up-to-date the Notice file for you. I suggest following edit to the generated file:

* shorten the list of items in 3rd party content, basically in this repo, we depend on asm only, so I think it should be fine to keep just that one

Reply:
and junit needs to stay in the list of 3rd party libs

Reply:
I'm not sure what I am supposed to do with https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.eclipselink. I navigate there and select "EclipseLink". Then it redirects me to my user profile... what is it supposed to do?

> keep junit

Which junit are you referring to (https://github.com/eclipse-ee4j/eclipselink/blob/master/NOTICE.md)? `pax-exam-junit4 (4.13.1)`?

Reply:
Ok, I got the tool working in IE. For some reason it doesnt work for me in firefox or chrome... 

> all our repos should be listed here

Ok, Ill put them all back, I just didn't think it made sense for THIS project. But maybe I am not thinking about the correct "project". Does "Project" refer to "eclipselink-asm" or "https://projects.eclipse.org/projects/ee4j.eclipselink"?

Reply:
in this case it refers to ee4j.eclipselink project, the owner of the eclipselink-asm (and others) repo(s)

wrt junit, I ment junit:junit:4.13.2 - the one which eclipselink-asm lib has defined in its pom/dependencies

Reply:
> I ment junit:junit:4.13.2

Interesting, I didn't see this in https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.eclipselink generated NOTICE.md. How would you like it to look?

Reply:
I added 
```
JUnit (4.13.2)

* License: Eclipse Public License
```
Hopefully, this works

Reply:
yes, I'll have to revisit this one after waking up. Too late here already ;-)